Examples/Python/GroomUtils.py

```diff
@@ -103,7 +103,7 @@ def applyCOMAlignment(outDir, inDataListSeg, inDataListImg, processRaw=False):
         T = img.center() - img.centerOfMass()
 
         # binarize result since linear interpolation makes image blurry again
-        img.translate(T).binarize().write(outname)
+        img.translate(T).binarize().recenter().write(outname)
```

Comment:
@jadie1 is this an image (CT/mri) , segmentation or antialiased segemenation? We should n't transform/resample a segmentation directly? we antialias it first.

Reply:
Okay. I didn't remove antialiasing, it was never in the groomutils COM alignment. I will add it!

Reply:
Added b0b2e9d